mmocr/models/textrecog/encoders/aster_encoder.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from mmcv.runner import Sequential
+
+from mmocr.models.builder import ENCODERS
+from mmocr.models.textrecog.layers import BidirectionalLSTM
+from .base_encoder import BaseEncoder
+
+
+@ENCODERS.register_module()
+class ASTEREncoder(BaseEncoder):
```

Comment:
@gaotongxiao  We can refactor this to LSTMEncoder. BiLSTM is not only used in ASTER, but also in other algorithms such as CRNN.